packages/vm/src/cache.rs

```diff
@@ -16,6 +16,9 @@ use crate::wasm_backend::{compile_and_use, compile_only, make_runtime_store};
 
 const WASM_DIR: &str = "wasm";
 const MODULES_DIR: &str = "modules";
+// Rounded-up average of Module size to optimized wasm size ratio
+// Based on `examples/module_size.sh`, and the (optimized) cosmwasm-plus modules
```

Comment:
I think we should see this as an upper bound instead of an average. If an attacker is would be able to get a factor of 70, the average does not change but you could force validator nodes to swap or even crash.

Reply:
OK, but, 70 will also be above our upper bound.

Reply:
Yeah, this remark is more about the comment and the strategy, not about the implementation.